spec/features/submitting_monthly_service_metrics_spec.rb

```diff
@@ -148,6 +148,46 @@
     expect(page).not_to have_content("Calls received should be the sum of the fields within 'Number of phone calls received, split by reasons for calling'")
   end
 
+  specify "Titles are not shown for non-applicable calls" do
+    svc = FactoryGirl.create(:service, name: 'No calls service', calls_received_applicable: false)
+    token = MonthlyServiceMetricsPublishToken.generate(service: svc, month: YearMonth.new(2017, 9))
+
+    visit publish_service_metrics_path(service_id: svc, year: '2017', month: '09', publish_token: token)
+    expect(page).not_to have_content('Total number of phone calls received')
```

Comment:
This can cause a bug as calls received can be non-applicable whilst the reasons for calling are still applicable. The user would run into the 'Calls received should be the sum of the fields within 'Number of phone calls received, split by reasons for calling' validation but wouldn't be able to submit the form as they wouldn't be able to update the calls received field due to it being non-applicable. Should we add a task to validate this on the services#edit page?

Reply:
I think you're right, I've created https://trello.com/c/qHySPfQG/320-validate-the-applicability-of-the-calls-metrics to track this.